plugins/identity-shiro/src/main/java/org/opensearch/identity/shiro/ShiroIdentityPlugin.java

```diff
@@ -26,7 +27,7 @@ public final class ShiroIdentityPlugin extends Plugin implements IdentityPlugin
     private Logger log = LogManager.getLogger(this.getClass());
 
     private final Settings settings;
-    private final AuthTokenHandler authTokenHandler;
+    private final ShiroTokenHandler authTokenHandler;
```

Comment:
So the way the interface will work is that each IdentityPlugin will be expected to conform to having its own TokenManager implementation. So since Shiro is an IdentityPlugin it has its own. The Security Plugin will also have its own. I swapped the name here because it seemed confusing when we had a native plugin that called something a generic name like that--its in the package but the name did not really tell you what it was a part of.